### ISSUE TITLE
memgpt load directory not working 

### ISSUE THREAD

**Issue by dividor** [cpacker/MemGPT#257]
**Describe the bug**
Loading data from a directory causes the error 

`ValueError: Must provide either 'input_dir' or 'input_files'`

Seems `memgpt load` requires 'input-dir' as an argument, but the underlying code requires 'input_dir'. 

**To Reproduce**
1. Create a directory with text files into a sub-folder called './data/json'
2. `memgpt load directory --name hdx --input-dir ./data/json`
3. See error

Here is the full error report:

```
memgpt % memgpt load directory --name hdx --input-dir ./data/json
╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│ /Users/<user>/opt/miniconda3/envs/appenv/lib/python3.11/site-packages/memgpt/cli/cli_load │
│ .py:34 in load_directory                                                                         │
│                                                                                                  │
│    31 │   │   │   recursive=True,                                                                │
│    32 │   │   )                                                                                  │
│    33 │   else:                                                                                  │
│ ❱  34 │   │   reader = SimpleDirectoryReader(input_files=input_files)                            │
│    35 │                                                                                          │
│    36 │   # load docs                                                                            │
│    37 │   print("Loading data...")                                                               │
│                                                                                                  │
│ ╭─────────────────────────────────────── locals ────────────────────────────────────────╮        │
│ │             get_index = <function get_index at 0x13e88b1a0>                           │        │
│ │             input_dir = './data/json'                                                 │        │
│ │           input_files = []                                                            │        │
│ │                  name = 'hdx'                                                         │        │
│ │             recursive = False                                                         │        │
│ │            save_index = <function save_index at 0x13e88bc40>                          │        │
│ │ SimpleDirectoryReader = <class 'llama_index.readers.file.base.SimpleDirectoryReader'> │        │
│ ╰───────────────────────────────────────────────────────────────────────────────────────╯        │
│                                                                                                  │
│ /Users/<user>/opt/miniconda3/envs/appenv/lib/python3.11/site-packages/llama_index/readers │
│ /file/base.py:112 in __init__                                                                    │
│                                                                                                  │
│   109 │   │   super().__init__()                                                                 │
│   110 │   │                                                                                      │
│   111 │   │   if not input_dir and not input_files:                                              │
│ ❱ 112 │   │   │   raise ValueError("Must provide either `input_dir` or `input_files`.")          │
│   113 │   │                                                                                      │
│   114 │   │   self.errors = errors                                                               │
│   115 │   │   self.encoding = encoding                                                           │
│                                                                                                  │
│ ╭─────────────────────────────────────────── locals ───────────────────────────────────────────╮ │
│ │        encoding = 'utf-8'                                                                    │ │
│ │          errors = 'ignore'                                                                   │ │
│ │         exclude = None                                                                       │ │
│ │  exclude_hidden = True                                                                       │ │
│ │  file_extractor = None                                                                       │ │
│ │   file_metadata = None                                                                       │ │
│ │  filename_as_id = False                                                                      │ │
│ │       input_dir = None                                                                       │ │
│ │     input_files = []                                                                         │ │
│ │ num_files_limit = None                                                                       │ │
│ │       recursive = False                                                                      │ │
│ │   required_exts = None                                                                       │ │
│ │            self = <llama_index.readers.file.base.SimpleDirectoryReader object at             │ │
│ │                   0x13ea25610>                                                               │ │
│ ╰──────────────────────────────────────────────────────────────────────────────────────────────╯ │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
ValueError: Must provide either `input_dir` or `input_files`.
```

**Expected behavior**
Files should be indexed, as noted in [the documentation](https://github.com/cpacker/MemGPT#data-sources-ie-chat-with-your-data).

**Screenshots**

**Additional context**

---

**How did you install MemGPT?**

- From the official package - `pip install pymemgpt`

**Your setup (please complete the following information)**

- Your OS:  MacOS
- Where you're trying to run MemGPT from: Terminal
- Your python version: 3.11.5
-  If you installed with `pip install pymemgpt`: 
  - `pymemgpt` version (run `pip list | grep pymemgpt`): 0.1.16

**Local LLM details**

I am configured for Azure OpenAI

**Comment by vivi** [comment]
For now, add `--recursive`, this will be patched in #246.

**Comment by dividor** [comment]
Thanks! Just saw the PR.
